docs/content/ingestion/compaction.md

```diff
@@ -77,7 +78,7 @@ An example of compaction task is
 
 This compaction task reads _all segments_ of the interval `2017-01-01/2018-01-01` and results in new segments.
 Since both `segmentGranularity` and `keepSegmentGranularity` are null, the original segment granularity will be remained and not changed after compaction.
-To control the number of result segments per time chunk, you can set `targetPartitionSize` or `numShards`. See [indexTuningConfig](../ingestion/native_tasks.html#tuningconfig) for more details.
+To control the number of result segments per time chunk, you can set `maxRowsPerSegment` or `numShards`. See [indexTuningConfig](../ingestion/native_tasks.html#tuningconfig) for more details.
```

Comment:
The same, there is no `maxRowsPerSegment` and  `numShards` for compaction tuningConfig. The `maxRowsPerSegment` is at the same level with `targetCompactionSizeBytes`, not in tuningConfig.